fiasco/level.py

```diff
@@ -55,6 +55,12 @@ def __init__(self, elvlc, wgfa):
         self._elvlc = elvlc
         self._wgfa = wgfa
 
+    def __getitem__(self, idx):
+        elvc = self._elvlc[idx]
```

Comment:
I don't think this is the right approach. The `_elvlc` and `_wgfa` properties are `DataIndexer` objects

https://github.com/wtbarnes/fiasco/blob/07f14fe1a5b4273bb2bb0743b612c6d31cd00632/fiasco/io/datalayer.py#L35

and their `__getitem__` is meant for accessing columns on those datasets, e.g. the 'E_obs', the observed energies on `_elvlc`.